content/en/docs/reference/rules/supported-fields/supported-fields.md

```diff
@@ -325,15 +275,46 @@ Name | Type | Description
 `fdlist.cports` | CHARBUF | for TCP/UDP FDs, for poll events, this is a comma-separated list of the client TCP/UDP ports in the 'fds' argument, returned as a string.
 `fdlist.sports` | CHARBUF | for poll events, this is a comma-separated list of the server TCP/UDP ports in the 'fds' argument, returned as a string.
 
-### Field Class: k8s
 
-Kubernetes context about pods and namespace name. These fields are populated with data gathered from the container runtime.
+### Field Class: container (plugin)
```

Comment:
Nice! Is this auto generated with Falco or did it require additional tweaks?

Reply:
Autogenerated!